packages/synapse-react-client/src/utils/functions/QueryFilterUtils.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  ColumnModel,
+  ColumnSingleValueFilterOperator,
+  ColumnSingleValueQueryFilter,
+  QueryFilter,
+  Row,
+} from '@sage-bionetworks/synapse-types'
+
+export function getValueFilter(
```

Comment:
Open to suggestions for a better name for this fn

Reply:
getPrimaryKeyINFilter() ?